glib/src/control_flow.rs

```diff
@@ -10,6 +10,8 @@ use crate::{prelude::*, translate::*};
 /// `ControlFlow::Continue` keeps the closure assigned, to be rerun when appropriate.
 ///
 /// `ControlFlow::Break` disconnects and drops it.
+///
+/// `Continue` and `Break` map to `true` and `false`, respectively.
```

Comment:
`G_SOURCE_CONTINUE` / `G_SOURCE_REMOVE` might be also helpful here?

Reply:
Are there similar ones for the signal propagation? There's `GDK_EVENT_PROPAGATE` / `GDK_EVENT_STOP`, those are the ones, right?

Reply:
Done.